components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -312,6 +316,21 @@ abstract class AbstractFetchDownloadService : Service() {
         }
     }
 
+    /**
+     * If notification accent color provided from browser side, returns it. Otherwise returns
+     * default accent color.
+     */
```

Comment:
We need a more generic docs for `Style`, as the current one is focused on `notificationAccentColor` but I think we could add more styling properties to `Style` :)

Reply:
Hmm, idk how to write proper docs. Could you please write down here docs, so i can use it :)

Reply:
We can write something like "Contains all the styles related to the download UI".

Reply:
Changed docs but it's not so good :/